site/gatsby-site/cypress/e2e/cite.cy.js

```diff
@@ -401,4 +401,12 @@ describe('Cite pages', () => {
       'Alleged: Tesla developed and deployed an AI system, which harmed Tesla.'
     );
   });
+
+  it('Should display response in timeline and as badge', () => {
+    cy.visit('/cite/51#r1765');
+
+    cy.get('[data-cy="responded-badge"]').should('exist');
+
+    cy.get('[data-cy="timeline-text-response"]').should('exist');
+  });
```

Comment:
This test looks good. In addition, we should have another test for an incident without responses and check that the Response badge and the green text in Timeline are not displayed in order to cover both scenarios.